### ISSUE TITLE
Accessibility: use of colour in cards

### ISSUE THREAD

**Issue by anatrofimenko** [rancher/ux#92]
When looking at accessibility per #4341 (rancher/dashboard#4324), we have a problem with colour usage is on “cards” with the top line colour-coded. This page does not provide additional information to explain the difference in card colours (the line on top that can be blue, yellow or red), only after opening the dropdown

![Image](https://github.com/user-attachments/assets/894ae8f5-477b-4c73-9888-7ba810cca22f)
![Image](https://github.com/user-attachments/assets/2cc0a5ce-f800-4ff8-8de7-716bde6b49a0)
Below, you will find a screenshot for quick reference:

Even if the dropdown was always open, the criteria would still fail since users who depend on SR would not be able to read the references.

Add an aria label with the description associated description to each of the cards. For example, a yellow card should have the aria-label=”Partner providers/services/cluster”. We recommend that there is more context than just the words “rancher”, “partners”, or “RKE2” so the user understands exactly to what they refer. This lines up with an overall suggestion to rely on an aria-label in each <div> to describe the category type for assistive technologies.

**Comment by anatrofimenko** [comment]
[Kenneth Wimer](https://jira.suse.com/secure/ViewProfile.jspa?name=kwwii) added a comment - [2023-12-04 16:30](https://jira.suse.com/browse/UXUI-184?focusedId=1311709&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1311709)
Existing redesign: https://xd.adobe.com/view/66c03ec6-d686-44f5-a6b6-ed33e6975f90-3393/
